community-containers/dlna/readme.md

```diff
@@ -0,0 +1,13 @@
+## DLNA server
+This container bundles DLNA server for your Nextcloud files to be accessible by the clients in your local network. Simply run the container and look for a new media server `nextcloud-aio` in your local network.
+
+### Notes
+- This is not working on Mac OS since it requires the `host` networking mode in docker, and it doesn't really share the host's network interfaces in this system
```

Comment:
```suggestion
- This is not working with Docker Desktop since it requires the `host` networking mode in docker, and it doesn't really share the host's network interfaces in this system
```